Octokit/Http/HttpClientAdapter.cs

```diff
@@ -83,25 +83,24 @@ protected virtual async Task<IResponse> BuildResponse(HttpResponseMessage respon
                 "application/x-gzip" ,
                 "application/octet-stream"};
 
-            using (var content = responseMessage.Content)
+            var content = responseMessage.Content;
+            if (content != null)
             {
-                if (content != null)
-                {
-                    contentType = GetContentMediaType(responseMessage.Content);
+                contentType = GetContentMediaType(content);
 
-                    if (contentType != null && (contentType.StartsWith("image/") || binaryContentTypes
+                if (contentType != null && (contentType.StartsWith("image/") || binaryContentTypes
                         .Any(item => item.Equals(contentType, StringComparison.OrdinalIgnoreCase))))
-                    {
-                        responseBody = await responseMessage.Content.ReadAsByteArrayAsync().ConfigureAwait(false);
```

Comment:
I agree with the preference of using Stream instead of ByteArray here. Given that this has changed would you mind updating the related code docs around the [Response](https://github.com/octokit/octokit.net/blob/03e7644f5ec2bbb9299d4531ca17a14c7569af60/Octokit/Http/Response.cs#L39C84-L39C104) implementation as well, just for clarity sake.

Reply:
Updated :)